test/test_webhook_rackspace.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
```

Comment:
Can we add an end-to-end test for rackspace as well? Something like `test_post_incident()` in `e2etest.py`. I think we're missing coverage on `rackspace.on_post()` and `alertmanager.on_post()` currently, which would explain the `super()` issue.